lago/prefix.py

```diff
@@ -104,6 +104,13 @@ def __init__(self, prefix):
         self.paths = paths.Paths(self._prefix)
         self._virt_env = None
         self._metadata = None
+        self._internal_repo = None
+
+    def _get_internal_repo(self):
+        return self._internal_repo
+
+    def _set_internal_repo(self, path):
```

Comment:
There's no need for getters and setters, this is python, use prefix.internal_repo instead (just make it public by convention)